src/views/webviews/welcome/welcome.css

```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: Arial, sans-serif;
+}
+
+header {
+  margin-top: 20px;
```

Comment:
should we be hardcoding pixels? What happens when we resize the window, etc?

Reply:
I have checked resizing of window, vscode web view automatically adjusts pixels based on it.